misc/images/materialized/Dockerfile

```diff
@@ -7,6 +7,37 @@
 # the Business Source License, use of this software will be governed
 # by the Apache License, Version 2.0.
 
+# 22.1.2 community images with amd and arm variants. Used here so we can copy
+# the cockroach binary if this is an arm host. We are temporarily using this
+# image from the community until an official image is published.
+FROM juliuszaromskis/cockroachdb:22.1.2 AS crdb-arm
```

Comment:
The arm and amd64 versions will be different - 22.1.2 vs. 22.1.5

Reply:
Yup :(. I think it is acceptable because there are no official cockroach arm images yet, but they have committed to producing them in the upcoming release in the fall, so for a few months we can deal with a community-built binary that is different than the official binary.